src/client/js/components/main.jsx

```diff
@@ -51,7 +51,7 @@ module.exports = React.createClass({
         {this.isThereAGameToday(nextGame)}
 
         <h2>{ this.props.homeTeam } vs. the fucking { nextGame.opponent }</h2>
-        <h3>{ moment(nextGame.date).format('h:mm a') } { moment(nextGame.date).format('dddd M/D/YYYY') } @ { nextGame.location}</h3>
+        <h3>{ moment(nextGame.time, 'h:ma Z').format('h:mm a') } { moment(nextGame.date).format('dddd M/D/YYYY') } @ { nextGame.location}</h3>
```

Comment:
The time in the data file is actually wrong. EDT is not correct. It seems as though all of the times were imported as mountain time. Stripping out the 'EDT' and just displaying the time would be correct. I checked this for their next away game as well and 12:20pm would be correct for mountain time when the game is in Chicago at 1:20pm. Not sure there is a way to get the data to be correct across multiple time zones since the data is wrong..

`nextGame.time.replace('EDT', '')`